contrib/eza/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "eza"
+pkgver = "0.17.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [
+    "cargo",
+    "pkgconf",
+]
+makedepends = [
+    "libgit2-devel",
+    "rust-std",
+    "zlib-devel",
+]
+pkgdesc = "Modern ls replacement"
```

Comment:
pkgdesc bad

Reply:
what would you like it to be

Reply:
make something up, but Subjective Adjectives shouldn't go in pkgdescs; pkgdesc should be neutral and descriptive ("Directory listing utility" or whatever)

Reply:
when they're that small it kinda sounds really weird for some reason, but okie

Reply:
i mean it's just as small as the previous one